ingestion_tools/scripts/data_validation/fixtures/data.py

```diff
@@ -0,0 +1,135 @@
+"""
+Any fixtures involving loading data from the bucket.
+"""
+
+import json
+from typing import Dict, List
+
+import mdocfile
+import ndjson
+import pandas as pd
+import pytest
+from data_validation_common.mrc import get_header
+from mrcfile.mrcinterpreter import MrcInterpreter
+
+from common.fs import FileSystemApi
+
+
+### Tilt series fixtures ###
+@pytest.fixture(scope="session")
+def tiltseries_mrc_headers(tiltseries_mrc_files: List[str]) -> Dict[str, MrcInterpreter]:
+    """Get the mrc file headers for a tilt series."""
+    headers = {}
+    for file in tiltseries_mrc_files:
+        try:
+            headers[file] = get_header(file)
+        except Exception as _:
+            pytest.fail(f"Failed to get header for {file}")
+
+    return headers
+
+
+@pytest.fixture(scope="session")
+def tiltseries_metadata(tiltseries_meta_file: str, filesystem: FileSystemApi) -> Dict:
```

Comment:
Could this just be metadata and shared across all metadata reads?

Reply:
tiltseries_metadata.json is different than tomogram_metadata.json and they do not share the same information that can be used by both tiltseries validation tests and tomogram validation tests? dataset_metadata.json doesn't have the information that tiltseries validation tests and tomogrma validation tests need either.